### PR TITLE
Patch for midstream

### PR DESCRIPTION
**Pull request by iranzo**
Adds patch for midstream URL vs upstream patch

### CODE REVIEW

**Comment by iranzo** [comment]
@ybettan

**Comment by ybettan** [comment]
/lgtm
@qbarrand PTAL

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 site_name: Kernel Module Management
 repo_url: https://github.com/rh-ecosystem-edge/kernel-module-management
 edit_uri: edit/main/docs/mkdocs
+site_description: Kernel Module Management (KMM) is a Kubernetes operator that manages, builds, signs and deploys out-of-tree kernel modules and device plugins on Kubernetes clusters.
```
Comment (qbarrand) [inline docs/mkdocs.yml:5]:
```suggestion
site_description: Kernel Module Management (KMM) is a operator that manages, builds, signs and deploys out-of-tree kernel modules and device plugins on OpenShift clusters.
```

**Comment by qbarrand** [comment]
/approve
@ybettan please re-lgtm

**Pull request merged**